pkg/controller/common/cleaner.go

```diff
@@ -168,6 +183,10 @@ func (c *cleaner) NeedLeaderElection() bool {
 	return true
 }
 
+func (c *cleaner) controllerName() string {
+	return fmt.Sprintf("%s_cleaner", c.name)
```

Comment:
the one place the cleaner is used already does append the cleaner suffix already. We can figure out a cleaner way of handling controller names in a subsequent pr, just documenting
https://github.com/Azure/aks-app-routing-operator/blob/1028b62d138483de6dfca64c54ed39c11c54c50f/pkg/controller/dns/external_dns.go#L46C37-L46C55

Reply:
In the interest of not being redundant - changed